internal/db/db.go

```diff
@@ -68,7 +69,7 @@ func createTables() {
 
 	_, err := db.Exec(createTableSQL)
 	if err != nil {
-		log.Fatalf("Ошибка при создании таблиц: %v", err)
+		log.Fatalf("Ошибка при создании таблиц: %w", err)
```

Comment:
Вот тут не оборачиваем, а просто логируем
```suggestion
		log.Fatalf("Ошибка при создании таблиц: %v", err)
```